src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -1900,6 +1906,27 @@ AssignTasksToConnectionsOrWorkerPool(DistributedExecution *execution)
 }
 
 
+/*
+ * SetNodeNameAndPort sets the nodename and nodeport for the given task placement
+ * with a lookup.
+ */
+static void
+SetNodeNameAndPort(ShardPlacement *taskPlacement, char **nodeName, int *nodePort)
```

Comment:
This feels more like a GetNodeNameAndPort

Reply:
LookupNodeNameAndPort maybe?

Reply:
LookupTaskPlacementAddress
LookupTaskPlacementHostAndPort